### ISSUE TITLE
Runtime error on try.openbiosim for 02_analysis_rbfe.ipynb

### ISSUE THREAD

**Issue by jmichel80** [OpenBioSim/BioSimSpaceTutorials#40]
When running the notebook 
[02_analysis_rbfe.ipynb](https://github.com/OpenBioSim/BioSimSpaceTutorials/blob/test-tryopenbiosim/04_fep/02_RBFE/02_analysis_rbfe.ipynb)

on try.openbiosim.org 

executing the following cell

```
results_all_files = []
no_repeats = 3
# create a list of the results files
for r in list(range(0, no_repeats)):
    file_name = f"analysis/freenrg_repeat_{r}_tyk2.csv"
    results_all_files.append(file_name)

results_files = []

for file in results_all_files:
    new_file_name = f"analysis/outputs/results_{results_all_files.index(file)}.csv"
    with open(new_file_name, "w") as result_file:
        writer = csv.writer(result_file, delimiter=",")
        writer.writerow(["lig_1", "lig_2", "freenrg", "error", "engine"])

        for row, index in pd.read_csv(file).iterrows():
            pert = f"{index['lig_1']}~{index['lig_2']}"
            if pert in perturbations:
                writer.writerow(
                    [
                        index["lig_1"],
                        index["lig_2"],
                        index["freenrg"],
                        index["error"],
                        index["engine"],
                    ]
                )

        results_files.append(new_file_name)
```

gives the following run time error

![image](https://github.com/OpenBioSim/BioSimSpaceTutorials/assets/11722212/f8a978bb-3b00-45b2-9ae2-621196009d27)

I see similar errors running cells below

![image](https://github.com/OpenBioSim/BioSimSpaceTutorials/assets/11722212/a5ea4482-f50f-4ef3-a63d-5e4b303009ce)

@annamherz  any ideas what is going on ?

**Comment by annamherz** [comment]
>                         index["freenrg"],
>                         index["error"],

I think in these lines with the updated files the new index names are `freenrg(kcal/mol)` and `error(kcal/mol)`. With this fixed, the next cell is then able to run without problems. Alternatively, instead of changing the notebook the headers in the file could be changed to only be `freenrg` and `error`, whichever is better?

**Comment by jmichel80** [comment]
closed by #41
